### DISCUSSION TITLE
bug or misunderstanding

### DISCUSSION THREAD

**Discussion by souri84** [GrapesJS/grapesjs#3982] [Q&A]
Hey everybody,

I would like to take an image for my custom block : 

There is a little loom : https://www.loom.com/share/66c589960afa4e928c7611b881a08b20

Thanks for your help :)

Thibaut

**Comment by souri84** [comment]
OK, I understand what is not working :

If my pointer is on `.gjs-block-label` element => OK but if my pointer is on `.gjs-block__media` element => NOK because builder think I want to drag and drop img and not custom html !

Loom explaination : https://www.loom.com/share/73262d0ef8724cedb45ff283bcd0e73d

**Comment by artf** [comment]
I'm able to see how D&D on images doesn't get the right placeholder preview (something I'm actually able to fix) but still, the dropped content on canvas is correct. Are you using the latest version? Are you able to create a reproducible demo?
